global.json

```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "5.0.301"
+        "version": "5.0.402"
```

Comment:
What is this needed for?

Reply:
Not actually necessary, but is there any reason _not_ to casually update to the latest minor version, with the latest patches?

Reply:
Nope, although I'd generally prefer updates like that be done in a separate PR so stuff isn't "snuck" in.

But this should be fine this time. Just try to avoid doing that in the future unless it's actually a part of the PR.

Reply:
Eh, nah you're right - I'll revert so that no one gets whiplash if either needs to be reverted.